core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -129,10 +130,11 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
     line-height: 0.6;
     padding-left: 4px;
     text-overflow: clip;
+    text-align: center;
```

Comment:
Hm, I'm not 100% sure about this. @markhalpin, two questions about the latest screenshot:

(a) Is the left edge of "AUTHOR PROFILES" in the right place?
(b) Is the left edge of "naveen" in the right place?

Thanks!